lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -222,7 +226,7 @@ bool RequirementSource::isInferredRequirement() const {
 }
 
 unsigned RequirementSource::classifyDiagKind() const {
```

Comment:
Errr... I was lazy. We could introduce an enum in `DiagnosticsSema` and use it for this.